arch/arm/src/stm32/stm32_rcc.c

```diff
@@ -207,15 +207,15 @@ void stm32_clockconfig(void)
 
 #endif
 
+  /* Enable peripheral clocking */
+
+  rcc_enableperipherals();
+  
```

Comment:
```suggestion
 
```
This whitespace is preventing a clean check.

Reply:
Sorry for this. Fixed bellow.

But please, how can I catch similar errors in the future?

Reply:
Thanks for fixing that.
You can run ./tools/checkpatch.sh on the files you changed.
`./tools/checkpatch.sh -f file`
Piping it with git you can run on all modified files:
`git diff --name-only | xargs ./tools/checkpatch.sh -f`
If the files have already been committed:
`git diff --name-only master..branchname | xargs ./tools/checkpatch.sh -f`